src/platforms/php/guides/laravel/configuration/capture.mdx

```diff
@@ -0,0 +1,255 @@
+
+In the following example, we'll use a middleware to add the user information if a user is logged in:
+
+```php
+namespace App\Http\Middleware;
+
+use Closure;
+use Sentry\State\Scope;
+
+class SentryContext
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @param \Closure                 $next
+     *
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if (auth()->check() && app()->bound('sentry')) {
+            \Sentry\configureScope(function (Scope $scope): void {
+                $scope->setUser([
+                    'id' => auth()->user()->id,
+                    'email' => auth()->user()->email,
+                ]);
+            });
+        }
+
+        return $next($request);
+    }
+}
+```
+
+### Log channels
+
+<Alert level="info" title="Note"><markdown>
+
+If you're using log channels to log your exceptions and are also logging exceptions to Sentry in your exception handler (as you would have configured above) exceptions might end up twice in Sentry.
+
+</markdown></Alert>
+
+Starting with Laravel & Lumen 5.6 we can setup Sentry using log channels.
+
+To configure Sentry as a log channel, add the following config to the `channels` section in `config/logging.php`:
+
+```php
+'channels' => [
+    // ...
+    'sentry' => [
+        'driver' => 'sentry',
+    ],
+],
+```
+
+After you configured the Sentry log channel, you can configure your app to both log to a log file and to Sentry by modifying the log stack:
+
+```php
+'channels' => [
+    'stack' => [
+        'driver' => 'stack',
+        // Add the Sentry log channel to the stack
+        'channels' => ['single', 'sentry'],
+    ],
+    //...
+],
+```
+
+Optionally, you can set the logging level and if events should bubble on the driver:
+
+```php
+'channels' => [
+    // ...
+    'sentry' => [
+        'driver' => 'sentry',
+        'level'  => null, // The minimum monolog logging level at which this handler will be triggered
+                          // For example: `\Monolog\Logger::ERROR`
+        'bubble' => true, // Whether the messages that are handled can bubble up the stack or not
+    ],
+],
+```
+
+### Queue jobs
+
+When you have defined a `failed` method on your job class ([documentation](https://laravel.com/docs/6.x/queues#cleaning-up-after-failed-jobs)) that failed method acts as if your job runs inside a `try {} catch (\Exception $e) {}` and this will prevent reporting exception causing the job to have failed to be reported to Sentry.
+
+This could be what you want since your job sometimes fails because of an API that is not reachable or other expected failures. If you still want the exception to be reported to Sentry you can do the following in your `failed` method:
```

Comment:
```suggestion
This could be expected since your job sometimes fails due to an API that is not reachable or other expected failures. If you still want the exception to be reported to Sentry, you can implement the following in your `failed` method:
```